x-pack/test/api_integration/apis/entity_manager/fixture_plugin/server/plugin.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+import {
+  Plugin,
+  CoreSetup,
+  RequestHandlerContext,
+  KibanaRequest,
+  KibanaResponseFactory,
+  Logger,
+  PluginInitializerContext,
+} from '@kbn/core/server';
+import { buildRouteValidationWithZod } from '@kbn/zod-helpers';
+import { upgradeBuiltInEntityDefinitions } from '@kbn/entityManager-plugin/server/lib/entities/upgrade_entity_definition';
+import { SecurityPluginStart } from '@kbn/security-plugin-types-server';
+import { EncryptedSavedObjectsPluginStart } from '@kbn/encrypted-saved-objects-plugin/server';
+import { entityDefinitionSchema } from '@kbn/entities-schema';
+
+interface FixtureStartDeps {
+  encryptedSavedObjects: EncryptedSavedObjectsPluginStart;
+  security: SecurityPluginStart;
+}
+
+export class FixturePlugin implements Plugin<void, void, {}, FixtureStartDeps> {
+  private logger: Logger;
+
+  constructor(context: PluginInitializerContext<{}>) {
+    this.logger = context.logger.get();
+  }
+
+  public setup(core: CoreSetup<FixtureStartDeps>) {
+    core.http.createRouter().post(
+      {
+        path: '/api/entities/upgrade_builtin_definitions',
+        validate: {
+          body: buildRouteValidationWithZod(
```

Comment:
Nit: You don't need to use `buildRouteValidationWithZod`, the Platform now has native support for zod schemas

Reply:
logged https://github.com/elastic/elastic-entity-model/issues/138